apps/web/composables/useActiveShippingCountries/useActiveShippingCountries.ts

```diff
@@ -1,5 +1,5 @@
 import type { ActiveShippingCountry } from '@plentymarkets/shop-api';
-import { useSdk } from '~/sdk';
+import { toRefs } from '@vueuse/shared';
```

Comment:
Nuxt handles auto-imports for it's own context, we can remove all manual imports (found in other files):
```import { toRefs } from '@vueuse/shared';```

Reply:
done